build/DevDivVsix/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.CommonFiles.swr

```diff
@@ -0,0 +1,21 @@
+use vs
+
+package name=Microsoft.VisualStudio.ProjectSystem.CommonFiles
+        version=2.0.0.0
+        vs.package.chip=neutral
+        vs.package.language=en-us
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft VisualStudio Managed Project System Common Files"
+                       description="Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects"
+
+folder "InstallDir:MSBuild\Microsoft\VisualStudio\v15.0\Managed"
```

Comment:
We can leave v15.0 out from the path I think. Doing that for existing files is hard because we have to update the callers but these are new files being dropped and since MSBuild will be app-local we don't need the 15.0.